assets/diagram-js.css

```diff
@@ -176,6 +176,20 @@ marker.djs-dragger text {
   pointer-events: none;
 }
 
+/**
+ * hit shape
+ */
+.djs-hit {
```

Comment:
We cannot do this, as it essentially prevents users from interacting with the viewer without the CSS file.

Reply:
OK